site/app/controllers/grading/ImagesController.php

```diff
@@ -4,51 +4,26 @@
 
 use app\controllers\AbstractController;
 use app\libraries\FileUtils;
+use app\models\DisplayImage;
 use app\models\User;
 use Symfony\Component\Routing\Annotation\Route;
 use app\libraries\Utils;
+use app\libraries\routers\AccessControl;
 
 class ImagesController extends AbstractController {
+
     /**
      * @Route("/courses/{_semester}/{_course}/student_photos")
      */
     public function viewImagesPage() {
```

Comment:
Instead of the PHP user group check you can use the `@AccessControl(role="STUDENT")` route annotation and avoid the extra code

Reply:
Changed